src/ir_def/check_ir.ml

```diff
@@ -724,10 +734,6 @@ let rec check_exp env (exp:Ir.exp) : unit =
   | SwitchE (exp1, cases) ->
     check_exp env exp1;
     let t1 = T.promote (typ exp1) in
-(*    if not env.pre then
```

Comment:
I believe we don't do coverage checking in the IR, right? This must be a leftover...